src/Checks/SolrIndexCheck.php

```diff
@@ -1,52 +0,0 @@
-<?php
```

Comment:
No point including this file since fulltextsearch is not and will not be CMS 6 compatible